vtds_cluster_kvm/private/config/config.yaml

```diff
@@ -224,6 +211,15 @@ cluster:
       # The 'node_count' field indicates how many nodes of this type
       # to instantiate on the cluster.
       node_count: 0
```

Comment:
REVIEWERS: half of the criterion for selecting a Node Builder and Disk Builder for a Virtual Node is knowing the Linux distribution family. This is added to the base configuration here.